### DISCUSSION TITLE
Heads up: we intend to archive the https://github.com/woocommerce/FlexSlider repository

### DISCUSSION THREAD

**Discussion by samueljseay** [woocommerce/woocommerce#48302] [Announcement]
Firstly, this is **not** a message about deprecating or removing flex slider functionality within WooCommerce! 😄 

This FlexSlider repository is out of date with the implementation we currently have within WooCommerce, it is not worked on actively, and it leads to occasional confusion when contributors reach out to raise bugs or ask questions. 

We will archive the repository in 30 days (on or after 10 July, 2024), but if anyone has any problems with this please let us know. If you ever need to contribute, discuss or raise bugs related to the use of flex slider in WooCommerce you can do it on the main https://github.com/woocommerce/woocommerce repository in future.

**Comment by amesplant** [comment]
@samueljseay 
What is the status on updating FlexSlider now that it is part of the WooCommerce main repo? We are working on accessibility issues that are getting reported, and there are a number related to FlexSlider. I'd like to propose either replacing FlexSlider with something more accessible, like SplideJS, or doing an overhaul on FlexSlider. An overhaul on FlexSlider would look like removing the jQuery dependency and making sure that it minimally meets WCAG 2.2 AA when it comes to accessibility.

cc @vikrampm1 @jeffpaul @dkotter

**Comment by samueljseay** [comment]
> What is the status on updating FlexSlider now that it is part of the WooCommerce main repo? We are working on accessibility issues that are getting reported, and there are a number related to FlexSlider. I'd like to propose either replacing FlexSlider with something more accessible, like SplideJS, or doing an overhaul on FlexSlider. An overhaul on FlexSlider would look like removing the jQuery dependency and making sure that it minimally meets WCAG 2.2 AA when it comes to accessibility

I couldn't agree more that it should be replaced or updated. I have not done the research on alternatives but removing the jQuery dependency would also be top of my list of what I'd want to see. 

I don't think anyone is prioritizing doing that work right now that I'm aware of, but we would welcome contributions to move this forward.

**Comment by vikrampm1** [comment]
Related accessibility issue: https://github.com/woocommerce/woocommerce/issues/31955
